src/applications/disability-benefits/all-claims/content/confirmation-page.jsx

```diff
@@ -8,17 +9,51 @@ import {
   errorMessage,
 } from '../content/confirmation-poll';
 
-const template = (props, title, content, submissionMessage) => {
+const template = (props, title, content, submissionMessage, messageType) => {
   const { fullName, disabilities, submittedAt } = props;
   const { first, last, middle, suffix } = fullName;
 
   const renderableContent =
     typeof content === 'string' ? <p>{content}</p> : content;
 
+  const alertBox = (
+    <AlertBox
+      isVisible
+      headline={title}
+      content={renderableContent}
+      status={messageType}
+    />
+  );
+
+  const backButtonContent = (
+    <div className="row form-progress-buttons schemaform-back-buttons">
+      <div className="small-6 usa-width-one-half medium-6 columns">
```

Comment:
IIRC, for most grid class utilities, doesn't `small-6` take care of the same styling as `medium-6` and is just inclusive of all screen sizes?

I'm also confused on the difference between those classes and the va design system ones and when to use one and not the other. might be overkill to bring up here, but we should try to work towards a single layout class system in any new code whenever possible